Language/Functions/Random Numbers/randomSeed.adoc

```diff
@@ -21,7 +21,7 @@ subCategories: [ "Random Numbers" ]
 
 If it is important for a sequence of values generated by `random()` to differ, on subsequent executions of a sketch, use randomSeed() to initialize the random number generator with a fairly random input, such as `analogRead()` on an unconnected pin.
 
-Conversely, it can occasionally be useful to use pseudo-random sequences that repeat exactly. This can be accomplished by calling `randomSeed()` with a fixed number, before starting the random sequence.
+Conversely, it can occasionally be useful to use pseudo-random sequences that repeat exactly. This can be accomplished by calling `randomSeed()` with a fixed number, before starting the random sequence. Note that if the `seed` is 0, `randomSeed(seed)` will have no effect.
```

Comment:
>Maybe it's useful to have a sort of warning banner/markup instead?

I agree. Please move this to a "**Notes and Warnings**" section of the page. You can use the source markup of the `random()` reference page as a model:

https://raw.githubusercontent.com/arduino/reference-en/master/Language/Functions/Random%20Numbers/random.adoc

Reply:
Good suggestion, thank you!